gammapy/modeling/models/spectral.py

```diff
@@ -1451,18 +1450,22 @@ class Absorption:
         Parameter node values
     data : `~astropy.units.Quantity`
         Model value
-    filename : str
-        Filename of the absorption model used for serialisation.
+    redshift : float
+        Redshift of the absorption model
+    alpha_norm: float
+        Norm of the EBL model
     interp_kwargs : dict
         Interpolation option passed to `ScaledRegularGridInterpolator`.
         By default the models are extrapolated outside the range. To prevent
         this and raise an error instead use interp_kwargs = {"extrapolate": False}
     """
 
-    tag = "Absorption"
+    tag = ["EBLAbsorptionNormSpectralModel", "ebl-absorption"]
+    alpha_norm = Parameter("alpha_norm", 1.0, frozen=True)
+    redshift = Parameter("redshift", 0.1, frozen=True)
 
-    def __init__(self, energy, param, data, filename=None, interp_kwargs=None):
-        self.filename = filename
+    def __init__(self, energy, param, data, redshift, alpha_norm, interp_kwargs=None):
+        self.filename = None
         # set values log centers
         self.param = param
```

Comment:
Is the `.param` still used? I think it's fixed to `redshift` now...

Reply:
I think we still need it to setup the ScaledRegularGridInterpolator and interpolate the table at the redshift value.

Reply:
Indeed you are right...